src/Pages/Settings.js

```diff
@@ -1,9 +1,21 @@
 import styled from "styled-components";
+import { useContext } from "react";
+import { AuthenticationContext } from "../Contexts/AuthenticationContenxt";
 
 function Settings() {
+  const { user, dispatch } = useContext(AuthenticationContext);
+
+  const onClickHandler = () => {
```

Comment:
please rename the function after the action that is performed on the click, e.g. handleSignOut

Reply:
Done